Sources/ApolloWebSocket/WebSocketTransport.swift

```diff
@@ -291,13 +296,14 @@ extension WebSocketTransport: WebSocketDelegate {
 
   public func websocketDidConnect(socket: WebSocketClient) {
     self.error.value = nil
-    initServer()
+    initServer(reconnect: reconnect.value)
```

Comment:
OK, I figured out where `reconnect.value` was being set before: In `initServer` at line ~193, where `reconnect` was defaulted to `true`. I think maybe if we can take out the loop of setting the reconnect value, that could help.